apps/studio.giselles.ai/services/teams/utils.ts

```diff
@@ -9,6 +9,22 @@ export function isProPlan(team: Pick<CurrentTeam, "plan">) {
 	);
 }
 
+const PLAN_LABELS: Record<string, string> = {
+	free: "Free plan",
+	pro: "Pro plan",
+	enterprise: "Enterprise plan",
+	internal: "Internal plan",
+};
+
+export function formatPlanName(plan: CurrentTeam["plan"]) {
+	return PLAN_LABELS[plan] ?? `${capitalize(plan)} plan`;
+}
+
+function capitalize(value: string) {
```

Comment:
You also can achieve it to use `titleCase`.

https://github.com/giselles-ai/giselle/blob/main/packages/utils/src/title-case.ts#L59-L68

Reply:
Thank you for your review!
Refactored with 18e7ec8a1c1e4e623c3ee6abc8259bc7e1496e47!!